lib/BackgroundJobs/StorageCrawlJob.php

```diff
@@ -65,6 +65,7 @@ protected function run($argument): void {
 				$this->diagnosticService->sendHeartbeat(static::class, $this->getId());
 				try {
 					$this->queue->insertIntoQueue($queueFile);
+					$i++;
```

Comment:
So, when the current batch of files all error , we don't schedule the job again?

Reply:
ah right, yeah we do but we don't want to update `last_indexed_file_id` in that case.